group_by.py

```diff
@@ -0,0 +1,26 @@
+from collections.abc import Iterable
+
+"""
+   The code will return a dictionary while:
+   The keys are the values returned from the function passed as the first parameter.
+   The value to a particular key is a list of all the organs for which the value appearing in the key is repeated.
+"""
+
+
+def group_by(func : callable , iterable : Iterable) -> dict:

```

Comment:
Func and iterable are not very goood argument names, func is abbreviated and too general and iterable can be easiliy confused with it's type.